tensorflow/lite/micro/tools/make/ext_libs/xtensa.inc

```diff
@@ -56,7 +56,7 @@ ifeq ($(TARGET_ARCH), hifi5)
     -I$(NNLIB_PATH)/algo/common/include/ \
     -I$(NNLIB_PATH)/algo/ndsp/hifi5/include/
 
-else ifeq ($(TARGET_ARCH), $(filter $(TARGET_ARCH), hifi4 hifi4_internal))
+else ifeq ($(TARGET_ARCH), $(filter $(TARGET_ARCH), hifi4))
```

Comment:
we can remove the filter. it can be `ifeq ($(TARGET_ARCH), hifi4)`

Reply:
Done. Thank you for the suggestion!